pom.xml

```diff
@@ -63,14 +63,15 @@ THE SOFTWARE.
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
-        <scm-api.version>2.0.7</scm-api.version>
+        <jenkins.version>2.60.3</jenkins.version>
```

Comment:
Avoidable with more work, but [who cares anyway](stats.jenkins.io/pluginversions/workflow-multibranch.html)?

Reply:
Yeah, we are going to be mandating 2.60.x here-on-out for Java 8 since the upstream plugins depend on it now anyway (and durability work means we'll need to bump to Java 8 versions).